src/js/preshuffle.s

```diff
@@ -756,17 +756,225 @@ MaybeSetCheckpoint:
 MaybeSetCheckpointActual:
 
 
+.segment "12", "13", "fe", "ff"
+;.bank $27000 $8000:$4000
+
+;; Overwrite the StatusBarDataTable with the new UI tile layout
+.org $badb
+  .byte $80,$20,$20,$20 ; Lf _ _ _
+.org $baee
+  .byte $83,$00,$00,$20 ; Lv 0 0 _
+.org $baf2
+  .byte $9c             ;  $
+
+.org $bafb
+  ;; on this row we shifted most everything left by three tiles
```

Comment:
maybe its because i spent so much time on researching this, that i actually find this kinda readable as is, but I'll try it out.

Reply:
I couldn't get the move to work, since its moving an overlapping region, the following `.org` call didn't seem to work and I'm really not planning on debugging this all night just to make it look a little nicer ^^